Public/Remove-PolarisRoute.ps1

```diff
@@ -0,0 +1,54 @@
+<#
+.SYNOPSIS
+    Removes the web route.
+.DESCRIPTION
+    Removes the web route(s) matching the specified path(s) and method(s).
+.PARAMETER Path
+    Path(s) of the route(s) to remove.
+    Accepts multiple values and wildcards.
+    Accepts pipeline input.
+    Accepts pipeline input by property name.
+    Defaults to all paths (*).
+.PARAMETER Method
+    Method(s) of the route(s) to remove.
+    Accepts pipeline input by property name.
+    Accepts multiple values and wildcards.
+    Defaults to all methods (*).
+.EXAMPLE
+    Remove-PolarisRoute
+    Removes all existing web routes.
+.EXAMPLE
+    Remove-PolarisRoute -Path 'helloworld' -Method GET
+    Removes the web route for method GET for path helloworld.
+.EXAMPLE
+    Remove-PolarisRoute -Path 'sub1/sub2/*'
+    Removes all web routes for all methods for paths starting with sub1/sub2/.
+.EXAMPLE
+    Remove-PolarisRoute -Path 'sub1/sub2/*' -Method GET, POST
+    Removes all web routes for GET and POST methods for paths starting with sub1/sub2/.
+.EXAMPLE
+    Get-PolarisRoute -Method Delete | Remove-Method
+    Removes all web routes for DELETE methods for all paths.
+#>
+function Remove-PolarisRoute {
```

Comment:
Nice to have: This function should take in a Polaris class object with a default value of $script:Polaris so that people can have multiple instances running in the same session.

Reply:
So... I added a Polaris class object with a default value to all of the scripts but it turned up an oddity with PowerShell classes. If you use them in the parameters of a function it will complain that it doesn't know about the [Polaris] type.

Apparently the parameter is validated in the user scope instead of the module scope and module classes aren't exported out of the module scope?

Reply:
wow. That's fascinating